src/views/ModelPlan/HomeTable/index.tsx

```diff
@@ -356,10 +356,12 @@ const ModelPlansTable = ({
           if (!value || value.length === 0) {
             return <div>{homeT('requestsTable.tbd')}</div>;
           }
-          const crtdlIDs = value
-            .map((crtdl: CRTDLType) => crtdl.idNumber)
-            .join(', ');
-          return crtdlIDs;
+
+          return value.map((crtdl: CRTDLType) => (
+            <ul>
+              <li>{crtdl.idNumber}</li>
+            </ul>
+          ));
```

Comment:
Spacing is off.

actual:
![image](https://github.com/user-attachments/assets/dfc32cfe-2de9-4282-983e-fc36d9aa6375)

expect:
![image](https://github.com/user-attachments/assets/c59dd2c7-fa19-4576-bc11-15c6697cee22)

Reply:
oh, while we are at it, the heading is different from figma. `FFS CRs and TDLs` vs figma's `CRs and TDLs`